src/main/java/edu/kis/powp/jobs2d/command/manager/parser/IDriverCommandParser.java

```diff
@@ -0,0 +1,11 @@
+package edu.kis.powp.jobs2d.command.manager.parser;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+import java.io.IOException;
+
+public interface IDriverCommandParser {
+    CommandDataModel parseFromFile(String data);
```

Comment:
Is data (param) a file (the name of the method suggests so)?!
Is it possible to convert directly to a DriverCommand? 
It would be easier to use by clients.

Reply:
In this case, we cannot do convert to DriverCommand.